test/test_torch.py

```diff
@@ -4600,6 +4570,36 @@ def test_masked_select_discontiguous(self, device):
                 torch.masked_select(v, m, out=out_dc)
                 self.assertEqual(out_dc, expected, atol=0, rtol=0)
 
+    @dtypes(*torch.testing.get_all_dtypes())
```

Comment:
you can get dtypes pairs using the syntax
```
 @dtypes(*product(torch.testing.get_all_dtypes(), (torch.uint8, torch.bool)))
 ```
 they are accessible inside the test via dtypes[0], dtypes[1]